perfkitbenchmarker/linux_benchmarks/cloudsuite_graph_analytics_benchmark.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs the graph analytics benchmark of Cloudsuite.
+
+More info: http://cloudsuite.ch/graphanalytics/
+"""
+
+import re
+
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import errors
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import sample
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.linux_packages import docker
+
+flags.DEFINE_string('worker_mem',
+                    '2',
+                    'Ammount of memory for the worker, in gigabytes,'
+                    'use 0 to ignore this option')
```

Comment:
So if I don't set `--executor-memory`, does it default to 2G? Or does it infer a number that could differ between machines?